src/client/replication_ddl_client.cpp

```diff
@@ -1350,6 +1354,8 @@ dsn::error_code replication_ddl_client::query_restore(int32_t restore_app_id, bo
     } else if (response.err == ERR_APP_DROPPED) {
         std::cout << "restore failed, because some partition's data is damaged on cold backup media"
                   << std::endl;
+    } else {
```

Comment:
Is it necessary to print some error message here ?

Reply:
`configuration_query_restore_response` type of struct doesn't have a `hint_message` like field, and the caller (in `https://github.com/apache/incubator-pegasus/blob/master/src/shell/commands/cold_backup.cpp#L493`) will print the error message if failed.
So it's not necessary to print error message here.

Reply:
OK.